desktop/cmp/impl/modalsupport/ModalSupport.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+
+import {box, fragment} from '@xh/hoist/cmp/layout';
+import {hoistCmp, uses, ModelPublishMode} from '@xh/hoist/core';
+import {dialog} from '@xh/hoist/kit/blueprint';
+import {Children} from 'react';
+import {createPortal} from 'react-dom';
+import {ModalSupportModel} from './ModalSupportModel';
+
+export const modalSupport = hoistCmp.factory({
+    displayName: 'ModalSupport',
+    model: uses(ModalSupportModel, {fromContext: false, publishMode: ModelPublishMode.NONE}),
+    render({model, children}) {
+        return fragment(
+            inlineContainer({model}),
+            modalContainer({model}),
+            createPortal(Children.only(children), model.hostNode)
+        );
+    }
+});
+
+const inlineContainer = hoistCmp.factory({
+    render({model}) {
+        return box({
+            ref: model.inlineRef,
+            height: '100%',
+            display: 'inherit',
+            flexDirection: 'inherit',
```

Comment:
I would like for both of these wrappers to have classnames - those really help me stay oriented to what is happening when I am looking at the resulting markup in Chrome inspector.

Would suggest `xh-modal-support xh-modal-support--inline`

Reply:
Sounds good- makes sense.